### PR TITLE
P1 rating

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

File: css/components/rating.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.rating-container {
+  float: left;
+  unicode-bidi: bidi-override;
+  direction: rtl;
+  font-size: 28px;
```
Comment (Megha-Joshi) [inline css/components/rating.css:5]: use rem instead of px

**Review by Megha-Joshi — APPROVED** [review]
Good to merge.

**Comment by sanjitsarkar** [comment]
Fix merge conflicts

File: css/components/rating.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  margin-top: -11px;
```
Comment (sanjitsarkar) [inline css/components/rating.css:6]: Use rem

**Review by sanjitsarkar — APPROVED** [review]

File: templates/rating.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <script src="https://kit.fontawesome.com/854a67f594.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <div class="container">
+      <div id="rating" class="rating-container prim-c ">
+  <span id="rating-first" title="5">☆</span>
+  <span id="rating-second" title="4">☆</span>
+  <span id="rating-third" title="3">☆</span>
+  <span id="rating-fourth" title="2">☆</span>
+  <span id="rating-fifth" title="1">☆</span>
```
Comment (pankaj-prog) [inline templates/rating.html:16]: check the title and id, are they supposed to be like this only

File: css/components/rating.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  margin-left: 10px;
+  margin-right: 10px;
+}
```
Comment (pankaj-prog) [inline css/components/rating.css:9]: use shorthand propery

File: css/components/rating.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
+.rating-container {
+  float: left;
```
Comment (pankaj-prog) [inline css/components/rating.css:2]: you can remove this. why do you need a float left on rating container

**Review by pankaj-prog — APPROVED** [review]

**Pull request merged**